src/gui/extras/30-envsubst-on-javascript.sh

```diff
@@ -19,8 +19,7 @@ vue_envsubst() {
   fi
 
   cd "$js_dir"
-  local defined_envs FILE
-  defined_envs=$(printf '${%s} ' $(env | fgrep VUE_ | cut -d= -f1))
+  defined_envs="$(printf '${%s} ' $(env | grep -F VUE_ | cut -d= -f1))"
```

Comment:
I'm not sure about security of this $env here ; could you verify?

Reply:
To exclude any potentially dangerous characters we could use something like:
```
env | grep -Eio '^VUE[-_a-z0-9]+'
```
That also obsoletes the `cut`.

But I don't like the complexity with the additional `printf`.

Reply:
I now committed the proposal